### DISCUSSION TITLE
Internal error after reboot

### DISCUSSION THREAD

**Discussion by openaspace** [modoboa/modoboa#2651] [Help]
This morning after server reboot (proxmox running omodoboa as container), web mail report **internal error**

**mail.err log report this:** 

Oct 19 21:42:52 mail dovecot: auth: Error: #011TCP/IP connections on port 5432? (state: connecting)
Oct 19 21:43:58 mail dovecot: auth: Error: pgsql(127.0.0.1): Connect failed to database modoboa: could not connect to server: Connection refused
Oct 19 21:43:58 mail dovecot: auth: Error: #011Is the server running on host "127.0.0.1" and accepting
Oct 19 21:43:58 mail dovecot: auth: Error: #011TCP/IP connections on port 5432? (state: connecting)
Oct 19 21:44:07 mail dovecot: auth: Error: pgsql(127.0.0.1): Connect failed to database modoboa: could not connect to server: Connection refused
Oct 19 21:44:07 mail dovecot: auth: Error: #011Is the server running on host "127.0.0.1" and accepting
Oct 19 21:44:07 mail dovecot: auth: Error: #011TCP/IP connections on port 5432? (state: connecting)
Oct 19 21:44:07 mail dovecot: auth: Error: pgsql(127.0.0.1): Connect failed to database modoboa: could not connect to server: Connection refused
Oct 19 21:44:07 mail dovecot: auth: Error: #011Is the server running on host "127.0.0.1" and accepting
Oct 19 21:44:07 mail dovecot: auth: Error: #011TCP/IP connections on port 5432? (state: connecting)

**SYSLOG**
Oct 20 12:07:06 mail postfix/cleanup[1296]: warning: 1C63B2F875: virtual_alias_maps map lookup problem for root@mail.domain.com -- message not accepted, try again later
Oct 20 12:07:07 mail postfix/pickup[1205]: 1CCCB2F97F: uid=113 from=<opendkim>
Oct 20 12:07:07 mail postfix/cleanup[1295]: warning: connect to Milter service inet:127.0.0.1:12345: Connection refused
Oct 20 12:07:07 mail postfix/cleanup[1295]: warning: proxy:pgsql:/etc/postfix/sql-aliases.cf lookup error for "opendkim@mail.domain.com"
Oct 20 12:07:07 mail postfix/cleanup[1295]: warning: 1CCCB2F97F: virtual_alias_maps map lookup problem for opendkim@mail.domain.com -- message not accepted, try again later
Oct 20 12:07:08 mail postfix/pickup[1205]: 1D1282FA80: uid=113 from=<opendkim>
Oct 20 12:07:08 mail postfix/cleanup[1296]: warning: connect to Milter service inet:127.0.0.1:12345: Connection refused
Oct 20 12:07:08 mail postfix/cleanup[1296]: warning: proxy:pgsql:/etc/postfix/sql-aliases.cf lookup error for "opendkim@mail.domain.com"
Oct 20 12:07:08 mail postfix/cleanup[1296]: warning: 1D1282FA80: virtual_alias_maps map lookup problem for opendkim@mail.domain.com -- message not accepted, try again later

**Comment by tonioo** [comment]
@openaspace Looks like your postgres server is down

**Reply by openaspace** [reply]
> @openaspace Looks like your postgres server is down

I'm quite estranged.
